app/api-v1/services/dscpApiService.js

```diff
@@ -19,7 +19,7 @@ const lastTokenId = async (authToken) => {
     return id
   }
 
-  throw new InternalError({ message: 'Internal server error', service: 'dscp-api' })
+  throw new InternalError({ message: 'Internal server error', res })
```

Comment:
`res` doesn't get used anywhere when errors are handled, we want to log it?

Reply:
did not see that this was an object, thought just a second argument